### ISSUE TITLE
rename ERD?

### ISSUE THREAD

**Issue by dimitri-yatsenko** [datajoint/datajoint-python#255]
@eywalker has suggested that we rename ERD into Graph to make it more friendly to new users.

So rather than 

``` python
dj.ERD(schema).draw()
```

we will use

``` python
dj.Graph(schema).draw()
```

**Comment by dimitri-yatsenko** [comment]
Or perhaps `Diagram`?

**Comment by eywalker** [comment]
Or maybe `Network`?

**Comment by dimitri-yatsenko** [comment]
I am afraid network implies web or tcp/ip connections.

**Comment by edgera** [comment]
I think that "graph" is more intuitive than "erd". Given our audience, my rank of the proposed terms:
"Graph" > "ERD" > "Diagram" > "Network"

**Comment by eywalker** [comment]
I'd also find `Graph` the most intuitive too. If we are going to make this change, I'd suggest that we do **not** try to make it back-compatible and just get rid of ERD (at least from Python side)

**Comment by dimitri-yatsenko** [comment]
If we keep things consistent in MATLAB and Python, then we should rename things in MATLAB too.  However, MATLAB treats things in the general name space differently from packages.  For example, if the name is in the general space, you can do this:
```matlab
erd experiment preprocess
```
which matlab interprets as `erd('experiment', 'preprocess')`.

However, if you put this inside a package, then this does not work:
```matlab
dj.erd experiment preprocess
```
will through an error.

Amazingly, with all its toolboxes, MATLAB keeps a lot of stuff in the common namespace.  The nice thing about `erd` is that it hasn't conflicted with any other name. 

Of course, we could completely get away from this syntax to be more consistent with what we did in Python.  Then we will see something like
```matlab
dj.Graph('experiment', 'preprocess').draw()
```
I like this solution but we will probably hear complains if we remove the old syntax.  It's okay if we are planning the first _real_ release.

**Comment by dimitri-yatsenko** [comment]
MATLAB introduced the graph object in R2015b.  We should update the implementation of dependencies to use the graph object.

**Comment by dimitri-yatsenko** [comment]
The graph plotting capabilities of MATLAB's `graph` class seem quite impressive.   We may be able to remove our own graph plotting functionality.

**Comment by fabiansinz** [comment]
I prefer `erd`. It's short, doesn't shadow existing libraries, and users will adapt it quickly in my opinion.

**Comment by dimitri-yatsenko** [comment]
I move that we close this issue and keep the object named `ERD`.

**Comment by eywalker** [comment]
I think I'm fine either way. It will be a matter of getting the Web GUI intuitive

**Comment by austin-hilberg** [comment]
@dimitri-yatsenko @eywalker We are considering changing ERD to Diagram.

**Comment by austin-hilberg** [comment]
This seems more appropriate now, since we have established that DataJoint is distinct from ERM.

**Comment by dimitri-yatsenko** [comment]
In release 0.12, `ERD` will be renamed into `Diagram` with `ERD` as an alias.

**Comment by dimitri-yatsenko** [comment]
Fixed in #565
